docs/tutorial/node/src/main_game_logic.js

```diff
@@ -42,7 +42,7 @@ mainWindow.check_if_pair_solved = function () {
             index: tile2_index
         } = flipped_tiles[1];
 
-        let is_pair_solved = tile1.image === tile2.image;
+        let is_pair_solved = tile1.image.path.path === tile2.image.path.path;
```

Comment:
Is that right `image.path.path`?

Reply:
Well spotted - thanks:)

Time to upgrade the tutorial to typescript:)

Reply:
Good idea. I was also thinking about to use TypeScript for our examples. What do you think?

Reply:
Absolutely yes :)

Reply:
Maybe we should have both?
Is there not some dev who prefer javascript?

Reply:
Yes, we can also have just JavaScript but still do type checking based at least on inferred types (so our QA). But generally I think we should promote static type systems.